docs/REST/authentication.md

```diff
@@ -289,3 +289,74 @@ static class HttpContentExtensions
     }
 }
 ```
+### Java
+```java
+public class ConfigHttpClient {
+
+    public CloseableHttpResponse execute(HttpUriRequest request, Date date, String credential, String secret) {
```

Comment:
Can we obtain the `date` in the `sign` function? A user doesn't have to care about it.